01-exercise-easy/07-point-counter/pointCounter_Dev.py

```diff
@@ -0,0 +1,14 @@
+def calcular_pontuacao(moedas):
+    cascalho = 0
+    for c in moedas:
```

Comment:
Visto que `C` é um tipo de moeda na task, não fica claro se adicionar `for c in moedas`.
 Poderia trocar o `C` pelo tipo = `moeda` ( que tipo de moeda das moedas do jogo o jogador recebe )
```suggestion
    for moeda in moedas:
```